### PR TITLE
new k8s guide

### PR DESCRIPTION
**Pull request by serverless-mom**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

Okay, so, this isn't a guide for Checkly, but... it didn't fit well in any /Learn category. Maybe I should have created it as a blog post?!? 😭 

Anyhoo, if we get a few more K8s articles for /Learn we can migrate this over there.

### CODE REVIEW

File: site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+author: Nočnica Mellifera
+avatar: 'images/avatars/nica-mellifera.png'
+tags:
+  - FAQ
+---
+We all implement tools to monitor our nodes and keep our entire cluster up and running. But how often do updates, failures, or errors mean that users suffer outages, even though our status boards look green? As Kubernetes has enabled more complex microservice architecture, the gap between the state of the dashboard, and the health of services for the user, has grown wider. Almost all serious Kubernetes teams need an external tool checking on their system, also known as a synthetic monitor, to really know that everything is working correctly for users. This article is a brief guide to two options: Kuberhealthy and Prometheus Blackbox exporter. The tools are quite different in cost, scope, maintenance, and execution so it’s no surprise they’ll fill quite different roles in ensuring uptime.
```
Comment (tnolet) [inline site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md:11]: this section is way too long and has no paragraph breaks, making it hard to read.

File: site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### Open source and self-hosted, for better or worse
+
+Prometheus Blackbox exporter is a completely free and open source tool, released under the Apache 2.0 license. That means no hidden fees, no surprise changes to SaaS terms, or the other vagaries that come with [adopting a tool like Cypress](https://www.checklyhq.com/blog/playwright-surpasses-cypress-for-downloads/#how-cypress-loses-to-playwright). However, this brings us to the greatest concern with relying too heavily on the Prometheus Blackbox exporter: the high probability it will be degraded at the same time as the rest of your service. Hopefully you’ve got your Prometheus instance in a separate cluster from your production systems, but even so it’s not certain that this service will survive a major outage. Further, there’s no inbuilt function for warning you about the lack of information. For example: an email that just says ‘We haven’t run a check with the blackbox exporter in over 30 minutes, something must be wrong!’
```
Comment (tnolet) [inline site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md:51]: Add paragraphs. This is a block of text.

File: site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- [Storage Provisioner Check](https://github.com/ChrisHirsch/kuberhealthy-storage-check) - verifies that a pod with persistent storage can be configured on every node in your cluster.
+
+---
+
+We can see some overlap with Prometheus Blackbox exporter, but clearly Kuberhealthy goes deeper than the Blackbox exporter. Making sure that pulls are possible of a particular remote repository is a more complex test than just seeing a response from an API endpoint. 
+
+However, notice what’s not included in this basic set of examples: nothing about checking a random URL’s API response and parsing the response. The focus here is on the health of components internal to your cluster. Kuberhealthy is likely to detect problems in your cluster, and those problems are very likely to be correlated to failures that are presented to your users. But these checks don’t answer questions like ‘Can users from X location log in’ or ‘Does our UI currently look correct to users?’ We can still use Kuberhealthy to detect problems on our service! But since we’re using the tool for something other than its core purpose, support and features may be hard to find.
```
Comment (tnolet) [inline site/content/guides/kuberhealthy-vs-prometheus-blackbox-exporter.md:88]: add paragraphs.

**Review by tnolet — CHANGES_REQUESTED** [review]
the content is hard to consume due to lack of paragraphs. Could also be more concise

**Review by Dgiordano33 — COMMENTED** [review]
This looks like general information for other tooling around k8s. To me, it feels like a good blog. 

A guide should use Checkly. A learn article should teach foundational skills. This feels like something for the blog where we give an opinion on something our readers are interested in.

**Comment by Dgiordano33** [comment]
@serverless-mom Could you just bring this to the blog and then close this PR down?

**Comment by serverless-mom** [comment]
Moved to blog here: https://www.checklyhq.com/blog/prometheus-blackbox-exporter-vs-kuberhealthy/

**Pull request closed without merging**